[2022-11-30T01:48:56.267+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2022-11-30T01:48:53.173250+00:00 [queued]>
[2022-11-30T01:48:56.280+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2022-11-30T01:48:53.173250+00:00 [queued]>
[2022-11-30T01:48:56.281+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T01:48:56.281+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2022-11-30T01:48:56.281+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T01:48:56.297+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-11-30 01:48:53.173250+00:00
[2022-11-30T01:48:56.302+0000] {standard_task_runner.py:55} INFO - Started process 2289 to run task
[2022-11-30T01:48:56.304+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'manual__2022-11-30T01:48:53.173250+00:00', '--job-id', '188', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpdyt63ou5']
[2022-11-30T01:48:56.306+0000] {standard_task_runner.py:83} INFO - Job 188: Subtask insert_into_table
[2022-11-30T01:48:56.379+0000] {task_command.py:376} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2022-11-30T01:48:53.173250+00:00 [running]> on host 2c9b3e546def
[2022-11-30T01:48:56.512+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bruno Lima
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2022-11-30T01:48:53.173250+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-30T01:48:53.173250+00:00
[2022-11-30T01:48:56.528+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-11-30T01:48:56.537+0000] {sql.py:315} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-11-30', 'dag_with_postgres_operator_v03')
        , parameters: None
[2022-11-30T01:48:56.541+0000] {sql.py:324} INFO - Rows affected: 1
[2022-11-30T01:48:56.555+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20221130T014853, start_date=20221130T014856, end_date=20221130T014856
[2022-11-30T01:48:56.604+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-30T01:48:56.627+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
